superset/assets/spec/javascripts/explore/components/ExploreViewContainer_spec.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
I think you didn't use `git mv` and git/github thinks you created a new file, let's keep the git history in place